web_sequence_fix/__init__.py

```diff
@@ -0,0 +1,21 @@
+# -*- encoding: utf-8 -*-
```

Comment:
Use coding instead of encoding and use short headers like https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py#L1-L3